website/content/docs/security/acl/acl-rules.mdx

```diff
@@ -493,41 +503,30 @@ node "admin" {
 }
 ```
 
-Node rules are segmented by the node name they apply to. In the example above, the rules allow read-only access to any node name with the empty prefix, allow
-read-write access to the "app" node, and deny all access to the "admin" node.
+#### Registering and Querying Node Information
 
-Agents need to be configured with an [`acl.tokens.agent`](/docs/agent/options#acl_tokens_agent)
-with at least "write" privileges to their own node name in order to register their information with
-the catalog, such as node metadata and tagged addresses. If this is configured incorrectly, the agent
-will print an error to the console when it tries to sync its state with the catalog.
+Agents must be configured with `write` or `read` privileges for their own node name so that the agent can register their node metadata, tagged addresses, and other information in the catalog. 
+If configured incorrectly, the agent will print an error to the console when it tries to sync its state with the catalog.
+Configure `write` or `read` access in the [`acl.tokens.agent`](/docs/agent/options#acl_tokens_agent) parameter.  
```

Comment:
I think the agent token requires `write`. Read won't allow it to sync all the data.